memlab.bib

```diff
@@ -28821,7 +28901,7 @@ @article{Schu55
 	Title = {Generalization of serial position in rote serial learning},
 	Volume = {49},
 	Year = {1955},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxAXLi4vRG9jdW1lbnRzL1NjaHU1NS5wZGZPEQFqAAAAAAFqAAIAAAtNYWNpbnRvc2hIRAAAAAAAAAAAAAAAAAAAAADDq+M4SCsAAACK4zwKU2NodTU1LnBkZgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJgToslHSXtQREYgQ0FSTwABAAIAAAkgAAAAAAAAAAAAAAAAAAAACURvY3VtZW50cwAAEAAIAADDrCmIAAAAEQAIAADJR4/LAAAAAQAMAIrjPACK4yUAAHmMAAIALU1hY2ludG9zaEhEOlVzZXJzOmtob3dlcjpEb2N1bWVudHM6U2NodTU1LnBkZgAADgAWAAoAUwBjAGgAdQA1ADUALgBwAGQAZgAPABgACwBNAGEAYwBpAG4AdABvAHMAaABIAEQAEgAhVXNlcnMva2hvd2VyL0RvY3VtZW50cy9TY2h1NTUucGRmAAATAAEvAAAVAAIADf//AAAACAANABoAJAA+AAAAAAAAAgEAAAAAAAAABQAAAAAAAAAAAAAAAAAAAaw=}}
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QFy4uL0RvY3VtZW50cy9TY2h1NTUucGRm0hcLGBlXTlMuZGF0YU8RAWoAAAAAAWoAAgAAC01hY2ludG9zaEhEAAAAAAAAAAAAAAAAAAAAAMOr4zhIKwAAAIrjPApTY2h1NTUucGRmAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAmBOiyUdJe1BERiBDQVJPAAEAAgAACSAAAAAAAAAAAAAAAAAAAAAJRG9jdW1lbnRzAAAQAAgAAMOsKYgAAAARAAgAAMlHj8sAAAABAAwAiuM8AIrjJQAAeYwAAgAtTWFjaW50b3NoSEQ6VXNlcnM6a2hvd2VyOkRvY3VtZW50czpTY2h1NTUucGRmAAAOABYACgBTAGMAaAB1ADUANQAuAHAAZABmAA8AGAALAE0AYQBjAGkAbgB0AG8AcwBoAEgARAASACFVc2Vycy9raG93ZXIvRG9jdW1lbnRzL1NjaHU1NS5wZGYAABMAAS8AABUAAgAN//8AAIAG0hscHR5aJGNsYXNzbmFtZVgkY2xhc3Nlc11OU011dGFibGVEYXRhox0fIFZOU0RhdGFYTlNPYmplY3TSGxwiI1xOU0RpY3Rpb25hcnmiIiBfEA9OU0tleWVkQXJjaGl2ZXLRJidUcm9vdIABAAgAEQAaACMALQAyADcAQABGAE0AVQBgAGcAagBsAG4AcQBzAHUAdwCEAI4AqACtALUCIwIlAioCNQI+AkwCUAJXAmACZQJyAnUChwKKAo8AAAAAAAACAQAAAAAAAAAoAAAAAAAAAAAAAAAAAAACkQ==}}
```

Comment:
Remove field

Reply:
This is a PDF of Schu55. Removed in latest commit.